coati/utils.py

```diff
@@ -1,18 +1,37 @@
 
-def grab_shape(element, shape_name):
-    return element.Shapes(shape_name)
+def grab_shape(slide, shape_name):
+    return slide.Shapes(shape_name)
 
 def grab_styles(shape):
     return {'width':  shape.Width,
             'height': shape.Height,
             'top':    shape.Top,
-            'left':   shape.Left}
+            'left':   shape.Left,
+            'lock_aspect_ratio': shape.LockAspectRatio}
 
 def apply_styles(shape, styles):
+    shape.LockAspectRatio = styles['lock_aspect_ratio']
     shape.Top    = styles['top']
     shape.Left   = styles['left']
     shape.Height = styles['height']
     shape.Width  = styles['width']
 
 def replace_text(label_shape, content):
     label_shape.TextFrame.TextRange.Text = content
+
+def transfer_properties(from_shape, to_shape):
+    apply_styles(to_shape, grab_styles(from_shape))
+    name = from_shape.Name
+    from_shape.Delete()
+    to_shape.Name = name
+
+def insert_image(slide, shape_name, filename, w = 4, h = 4):
```

Comment:
Can we remove insert_image from utils?
@lgovea already created a class to do this